### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hodaknight**
it's me hoda

### CODE REVIEW

**Comment by hodaknight** [comment]
1

**Comment by nvhdanh** [comment]
2

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# pr
\ No newline at end of file
+# pr
```
Comment (nvhdanh) [inline README.md:1]: this is pr

**Review by nvhdanh — COMMENTED** [review]
comment

**Comment by nvhdanh** [comment]
comment

**Review by nvhdanh — COMMENTED** [review]
comment 2

**Review by nvhdanh — APPROVED** [review]
Approve

**Pull request merged**